components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -629,9 +642,13 @@ void FeedV2Builder::BuildFollowingFeed(
       subscriptions, {.signals = true},
       mojom::FeedV2Type::NewFollowing(mojom::FeedV2FollowingType::New()),
       base::BindOnce([](FeedGenerationInfo info) {
-        return GenerateBasicFeed(std::move(info),
-                                 base::BindRepeating(&PickRoulette),
-                                 base::BindRepeating(&PickRoulette));
+        auto subscriptions = info.subscriptions();
+        auto top_stories = GetTopStoryUrls(info.topics());
+        return GenerateBasicFeed(
+            std::move(info), base::BindRepeating(&PickRoulette),
+            base::BindRepeating(&PickRoulette),
+            base::BindRepeating(&PickPeekingCard, std::move(subscriptions),
+                                std::move(top_stories)));
```

Comment:
nit: I think passing rvalue like this would be same instead of having temp vars(`subscriptions` and `top_stories`)?
```suggestion
            base::BindRepeating(&PickPeekingCard, info.subscriptions,
                                GetTopStoryUrls(info.topics())));
```

Reply:
Unfortunately we can't, as we move `info` into the callback, so it'd be undefined.